modules/ROOT/pages/apikit-error-handling-reference.adoc

```diff
@@ -1,38 +1,64 @@
-= APIkit Error Handling Reference
+= Error Handling Reference
```

Comment:
I think you need the product name for SEO.

Reply:
OK! Thanks!